azure-pipelines.yml

```diff
@@ -0,0 +1,38 @@
+strategy:
+  matrix:
+    linux:
+      imageName: 'ubuntu-16.04'
+    mac:
+      imageName: 'macOS-10.14'
+    windows:
+      imageName: 'vs2017-win2016'
+
+pr:
+  - master
+
+pool:
+  vmImage: $(imageName)
+
+steps:
+  - task: NodeTool@0
+    inputs:
+      versionSpec: '12.x'
+    displayName: 'Install Node.js'
+
+  - script: yarn
+    displayName: 'Install deps'
+
+  - script: npx jest
+    displayName: 'Run tests'
+
+  - script: yarn build
+    displayName: 'Build the code'
+
+  - script: npx pkg --output ./cli-binaries/prism-cli-linux ./packages/cli/
+    displayName: 'Create binary'
+
+  - script: yarn test.harness
+    displayName: 'Run harness'
+
+  - publish: ./cli-binaries/prism-cli-linux
```

Comment:
Do you  mean to be testing the Linux ones on azure?

Reply:
Well it turns out it's free anyway — I can remove it though, it does not add that much value effectively.